app/controllers/state_file/questions/final_income_review_controller.rb

```diff
@@ -0,0 +1,71 @@
+module StateFile
+  module Questions
+    class FinalIncomeReviewController < QuestionsController
+      include ReturnToReviewConcern
+      before_action :set_sorted_vars
+
+      def self.show?(intake)
+        intake.state_file_w2s.present? ||
+          intake.direct_file_data.fed_unemployment.positive? ||
+          intake.state_file1099_rs.present? ||
+          intake.direct_file_json_data.interest_reports.count.positive? ||
+          intake.direct_file_data.fed_ssb.positive? || intake.direct_file_data.fed_taxable_ssb.positive?
+      end
```

Comment:
should pull out shared methods into a shared controller that these inherit from with different `update` methods